static/app/views/replays/list.tsx

```diff
@@ -35,10 +49,22 @@ function ReplaysListContainer() {
         <Layout.Body>
           <Layout.Main fullWidth>
             <LayoutGap>
-              <ReplaysFilters />
-              <ReplaysErroneousDeadRageCards />
-              <ReplaysSearch />
-              <ReplaysList />
+              <ReplayListPageHeaderHook />
+              {fetching ? null : showOnboarding ? (
```

Comment:
Should we show a loader?

Reply:
I couldn't get it to show the wrong thing here. I think the null is mostly about preventing a flash-of-wrong content.
idk